aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -933,6 +934,47 @@ Tensor _test_serialization_subcmul(const Tensor& self, const Tensor& other, Scal
   return self - (other * alpha);
 }
 
+Tensor& heaviside_out(Tensor& result, const Tensor& self, const Tensor& val) {
+  TORCH_CHECK(!self.is_complex() && !result.is_complex() && !val.is_complex(),
+              "heaviside is not implemented for complex tensors.");
```

Comment:
"not" -> "not yet"

to be consistent with the following statement (also because heaviside could be implemented for complex)

Reply:
Addressed!